crates/router/src/connector/wise.rs

```diff
@@ -589,6 +589,17 @@ impl
         types::PayoutsResponseData,
     > for Wise
 {
+    fn build_request(
+        &self,
+        _req: &types::PayoutsRouterData<api::PoEligibility>,
+        _connectors: &settings::Connectors,
+    ) -> CustomResult<Option<services::Request>, errors::ConnectorError> {
+        // Eligiblity check for cards is not implemented
+        Err(errors::ConnectorError::NotImplemented(
+            "Payout Cards for Wise".to_string(),
+        ))
+        .into_report()?
+    }
```

Comment:
We can convert `ConnectorError` to `Report` type

```suggestion
    fn build_request(
        &self,
        _req: &types::PayoutsRouterData<api::PoEligibility>,
        _connectors: &settings::Connectors,
    ) -> CustomResult<Option<services::Request>, errors::ConnectorError> {
        // Eligiblity check for cards is not implemented
        Err(errors::ConnectorError::NotImplemented("Payout Cards for Wise".to_string()).into())
    }
```